package.json

```diff
@@ -22,14 +22,14 @@
   "scripts": {
     "lint": "eslint crawler/**.js dashboard/**.js dataStore/**.js scripts/**.js test/**.js",
     "build": "npm-run-all build:**",
-      "build:scripts": "babel scripts -d dist/scripts",
-      "build:crawler": "babel crawler -d dist/crawler",
-      "build:dashboard:js": "babel dashboard/js -d dist/dashboard",
-      "build:dashboard:static": "shjs dist/scripts/build-dashboard-static.js",
-      "build:datastore": "babel dataStore -d dist/dataStore",
+    "build:scripts": "babel scripts -d dist/scripts",
+    "build:crawler": "babel crawler -d dist/crawler",
+    "build:dashboard:js": "babel dashboard/js -d dist/dashboard",
+    "build:dashboard:static": "shjs dist/scripts/build-dashboard-static.js",
+    "build:datastore": "babel dataStore -d dist/dataStore",
```

Comment:
nit: Please fix the tabbing that got removed here... it happens every time we add an npm module :-/